hooks/use-deposit.ts

```diff
@@ -24,21 +21,42 @@ type DepositDataJson = {
   fork_version: string;
 };
 
+const GET_DEPOSIT_EVENTS = gql`
+  query MyQuery($pubkeys: [String!], $chainId: Int!) {
+    SBCDepositContract_DepositEvent(
+      limit: 10, 
```

Comment:
Let's try to see how we can make the query without the limitation and check if there is a maximum query limit (locally and on hosted service). In that case we might need to implement the same approach as in subgraph to make sure we get all deposit events

Reply:
oops, it had to be without `limit`, I've just copied from envio console and forgot to delete the line 😅 Good catch, thank you @Wagalidoom!

I think there's no need in limits, bcs how many pubkeys can be in the json file? ok, maybe if it's more than 1000 makes sense to split the queries.